magenta/music/musicxml_reader.py

```diff
@@ -0,0 +1,116 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MusicXML import.
+
+Input wrappers for converting MusicXML into tensorflow.magenta.NoteSequence.
+"""
+
+# internal imports
+from magenta.music.musicxml_parser import MusicXMLDocument
+from magenta.protobuf import music_pb2
+# pylint: enable=g-import-not-at-top
```

Comment:
I'm wondering why the pylint comment is needed. I don't see any imports out of place, so there shouldn't be any need to disable that linter check.